praetorian_cli/ui/console/commands/tools.py

```
@@ -0,0 +1,420 @@
+            try:
+                login_user = self.sdk.accounts.login_principal()
+            except Exception:
+                pass
+            display_user = login_user or 'API key owner'
+
+            self.console.print(f'[dim]Account frozen -- retrying as {display_user}[/dim]')
+            saved = self.sdk.keychain.account
+            self.sdk.keychain.account = None
+            try:
+                result = self.sdk.jobs.add(target_key, [capability], config_str)
+            except Exception as e2:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)